crates/mithril-oxide-sys/src/InitAllTranslations.rs

```diff
@@ -0,0 +1,21 @@
+pub use self::ffi::registerAllTranslations;
+pub use self::ffi::registerFromLLVMIRTranslation;
+pub use self::ffi::registerFromSPIRVTranslation;
+pub use self::ffi::registerToCppTranslation;
+pub use self::ffi::registerToLLVMIRTranslation;
+pub use self::ffi::registerToSPIRVTranslation;
```

Comment:
Maybe you could combine them into a single statement?
Like this:

```suggestion
pub use self::ffi::{
    registerAllTranslations,
    registerFromLLVMIRTranslation,
    registerFromSPIRVTranslation,
    registerToCppTranslation,
    registerToLLVMIRTranslation,
    registerToSPIRVTranslation,
};
```